packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -108,54 +188,60 @@ export class TreeConfiguration<TSchema extends ImplicitFieldSchema = ImplicitFie
 		public readonly initialTree: () => InsertableTreeFieldFromImplicitField<TSchema>,
 		options?: ITreeConfigurationOptions,
 	) {
-		this.options = { ...defaultTreeConfigurationOptions, ...options };
+		this.enableSchemaValidation =
+			options?.enableSchemaValidation ??
+			defaultTreeConfigurationOptions.enableSchemaValidation;
 	}
 }
 
 /**
- * An editable view of a (version control style) branch of a shared tree.
+ * An editable view of a branch of a shared tree based on some schema.
```

Comment:
The word "branch" is confusing when applied to trees when not talking about tree branches. This sounds very much like it could be applied to a subtree kind of branch, especially given that applying these kinds of configurations to subtrees is a feature we may want someday.

I think the clarification on what is meant by branch has value here, and I don't see much of a gain to removing it.

Remember than branches are not currently a concept in our public API, so its not a well defined term (and if it was, it would need to be defined here any way)